oemoflex/tools/plots.py

```diff
@@ -179,6 +246,124 @@ def filter_timeseries(df, start_date=None, end_date=None):
     return df_filtered
 
 
+def assign_stackgroup(key, values):
+    r"""
+    This function decides if data is supposed to be plotted on the positive or negative side of the
+    stackplot. If values has both negative and positive values, a value error is raised.
+
+    Parameters
+    ---------------
+    key : string
+        Column name.
+    values: pandas.Series
+        Values of column.
+
+    Returns
+    ----------
+    stackgroup : string
+        String with keyword positive or negative.
+    """
+    if all(values <= 0):
+        stackgroup = "negative"
+    elif all(values >= 0):
+        stackgroup = "positive"
+    elif all(values == 0):
+        stackgroup = "positive"
+    else:
+        raise ValueError(
+            key,
+            " has both, negative and positive values. But it should only have either one",
+        )
+
+    return stackgroup
+
+
+def plot_dispatch_plotly(
```

Comment:
Should we allow to pass a custom mapping as well?

Reply:
Yes, might be a good idea to make it more accessable.

Reply:
Will be done in a new PR.